src/edge-cs-coreclr/global.json

```diff
@@ -0,0 +1,7 @@
+{
+    "sdk": {
+        "version": "1.0.0-beta7-12331",
+        "runtime": "coreclr",
+        "architecture": "x64"
```

Comment:
What is the implication of the architecture choice here? Is this going to work with x86 edge after compilation (eventually, after that CLR bug is fixed).